### DISCUSSION TITLE
Commenting is not a feature for self hosted type?

### DISCUSSION THREAD

**Discussion by kost2191** [outline/outline#5215] [Self Hosting] [answered]
Hi everyone, 
In changelog it says that in march 23 comments are released, but the last release doesn't consist it (0.68.1). I see commits for this new feature in main branch. When are you going to make new stable release?

**Comment by tommoor** [comment, marked as answer]
It hasn't been released for community yet, the changelog is only applicable for cloud customers – for community edition release notes GitHub is the place to look: https://github.com/outline/outline/releases

The next release will be soon, in the next week or so.

**Reply by kost2191** [reply]
thx, built myself from main now for tests!
